app/models/hyrax/collection_type.rb

```diff
@@ -109,5 +109,18 @@ def ensure_no_collections
         errors[:base] << I18n.t('hyrax.admin.collection_types.error_not_empty')
         throw :abort
       end
+
+      def ensure_no_settings_changes
+        return true unless collections?
+        return true unless
```

Comment:
The indenting looks a bit funky here. I would consider reworking this as follows:

```ruby
def ensure_no_settings_changes
  # ...
  return true if collection_type_properties_changed?
  errors[:base] << ...
  # ...
end

def collection_type_properties_changed?
  ['nestable', 'discoverable', 'sharable', 'allow_multiple_membership', 'require_membership', 'assigns_workflow', 'assigns_visibility'].any? do |key|
     key.in?(changes)
  end
end
```